css/secondary-page.styl

```diff
@@ -55,11 +55,25 @@
         margin-bottom: 1em
 
       .hero-nav
+        ul
+          display: flex
+          flex-wrap: wrap
+          margin: 0
+          padding: 0
+          list-style-type: none
+        li
+          margin: 0
+          padding: 0
+
         a
           @extends .main-nav-item
           font-size: 14px
           line-height: 3em
           text-decoration: none
+          &:focus
```

Comment:
Could you also apply these styles for hover? `&:focus, &:hover`

Reply:
Done. :) Sorry for the late reply. I've been trekking around the Colombian jungle. ;)

Reply:
No worries! I was on holiday last week too!